odk1-src/collect-settings.rst

```diff
@@ -183,6 +183,18 @@ To access Form Management settings:
 
 .. rubric:: Form update
 
+:guilabel:`Blank form update mode`
+"""""""""""""""""""""""""""""""""""
+
+  Specifies how blank forms should be updated:
+
+  :guilabel:`Manually`
+    Enumerators will have to manually manage blank forms on the device using :guilabel:`Get Blank Form` and :guilabel:`Delete Saved Form`.
+  :guilabel:`Previously downloaded forms only`
+    The enumerator will receive a notification when one or more forms already downloaded have an update available to their form definition or media files. Tapping on the notification will go to :guilabel:`Get Blank Form` where the user can choose to download some or all of the updated forms.
```

Comment:
`The enumerator` -> `Enumerators`

`forms already downloaded` -> `forms on the device`